### ISSUE TITLE
Crash when assigning items to duplicants

### ISSUE THREAD

**Issue by MG665** [onimp/oni_multiplayer#146]
As the title implies.
```
NullReferenceException: Object reference not set to an instance of an object

MultiplayerMod.Multiplayer.Commands.Gameplay.Assignables.Assign.Execute () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator+<>c__DisplayClass12_0.<ClientOnCommandReceived>b__1 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl+<>c__DisplayClass1_0.<RunWithDisabledPatches>b__0 () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches[T] (System.Func`1[TResult] action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Core.Patch.PatchControl.RunWithDisabledPatches (System.Action action) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Multiplayer.Configuration.MultiplayerCoordinator.ClientOnCommandReceived (System.Object sender, MultiplayerMod.Network.Events.CommandReceivedEventArgs e) (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.ReceiveCommands () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.SteamClient.Tick () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)
MultiplayerMod.Platform.Steam.Network.Components.SteamClientComponent.Update () (at <f073262436b54d6ab22e3ff4a3e50ace>:0)

Build: U47-562984-V
```

**Comment by polycone** [comment]
@MG665 Hi there, were these items created in a multiplayer session? If that's the case we haven't implemented buildings id sync yet (#133). But we agreed the game shouldn't crash in that case.

**Comment by polycone** [comment]
Crash is fixed in #151.
Sync will be implemented in #133.

**Comment by MG665** [comment]
> @MG665 Hi there, were these items created in a multiplayer session? If that's the case we haven't implemented buildings id sync yet (#133). But we agreed the game shouldn't crash in that case.

Thank you for the replay and recent fixes. Yes we use this mod solely for multiplayer and all reports/crashes are from multiplayer sessions.

**Comment by polycone** [comment]
Yeah, sorry I didn't provide more context 😅, by "created in a multiplayer session" I meant that objects weren't loaded on client full sync (i.e. from an actual save file) that occurs on join / new cycle, but were created in a running game by player actions. Anyway I've checked and I think that was the case. Thanks for the report!
